pixl_dcmd/src/pixl_dcmd/main.py

```diff
@@ -216,3 +219,37 @@ def _whitelist_tag(dataset: Dataset, de: DataElement, tag_scheme: list[dict]) ->
     ]["op"] != "delete":
         return
     del dataset[de.tag]
+
+
+# NOTE: do we want to make the standard configurable per project?
+def check_valid_dicom(dataset: Dataset) -> bool:
+    """Validate the DICOM dataset against the given standard."""
+
+    # Default from dicom_validator but defining here to be explicit
+    standard_path = str(Path.home() / "dicom-validator")
+    revision = "current"
+
+    # dicom_validator doesn't have a dedicated API, so setting up the components ourselves. Taken from
+    # https://github.com/pydicom/dicom-validator/blob/37a5f9770ae8f44fa6fb2aaa6bfed57743e01896/dicom_validator/validate_iods.py
+    edition_reader = EditionReader(standard_path)
+    destination = edition_reader.get_revision(revision, False)
+    json_path = Path(destination, "json")
+    dicom_info = EditionReader.load_dicom_info(json_path)
+
+    # NOTE: dicom_validator returns a dict with all deviations from the standard
+    # Currently we're only letting datasets pass that don't have any exceptions
+    # This is likely to stringent for most applications, so we might want to parse the results more
```

Comment:
Would be interesting to see if this is the case on some live data. I'd expect we want to be fully compliant with DICOM spec. Could use this as a way to onboard onto running on the GAE as well
```suggestion
    # This is likely too stringent for most applications, so we might want to parse the results more
```